### PR TITLE
text overlapping problem solved

### LINKED ISSUE #401: Overlapping text
## **Describe the bug**

- A clear and concise description of what the bug is.
- Text overlaps in mobile view

## **Possible solution**

- Describe the solution you thought of.
- To make text responsive using bootstrap.

## **Screenshots**

- If applicable, add screenshots to help explain your problem.
Screenshot (497)

### PR DESCRIPTION
**Pull request by sreetu**
# Description

Please include a summary of the change and which issue is fixed.
I have made the ".banner-text" class responsive so that the text in that section does not overlap with the About section.
Fixes #401

## List any dependencies that are required for this change 
none
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
On my local host

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
Screenshot (497)

    - After :
Screenshot (623)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by sreetu** [comment]
Requesting @admins to review and merge my PR
Screenshot (626)

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Still it is being overlapped in 320px resolution.
image

**Comment by sreetu** [comment]
Sir, I have made the changes in the css file.

File: css/style.css [hunk 1]
```diff
@@ -186,8 +186,9 @@ header {
   font-weight: bold;
 }
 
-.below-head {
-  margin-left: 3vw;
+.banner-text .below-head {
+  margin: 50px 0 0 3vw;
+  font-size: 3.2vw;
```
Comment (harikesh409) [inline css/style.css:191]: On keeping this size it looks huge and there is no much differentiation between the "We are programmers" and the subtitle and even the alignment looks a bit off, try to adjust it.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Review by ravikr126 — APPROVED** [review]

File: css/style.css (hunk 1 above)
Comment (sreetu) [reply css/style.css:191]: Ok Sir

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**